build/build_standalone_script.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Fichier de build du script. Il prend tous les fichiers dans build/install_scripts/, itère dessus et vient les injecter dans le script.sh.